src/main.jsx

```diff
@@ -1,23 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { GameEngineProvider, GlobalStyles } from './shared';
-import { GameUI } from './ui';
+import { GameUI, HomePage, LevelSelectPage, NotFound } from './ui';
 import './public/index.css';
 
 const App = () => {
-  return (
-    <>
-      <GlobalStyles />
-      <GameEngineProvider>
-        <GameUI />
-      </GameEngineProvider>
-    </>
-  );
+  return <HomePage />;
 };
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <GlobalStyles />
+    <BrowserRouter>
+      <Routes>
+        <Route path='/' element={<App />} />
+        <Route
+          path='game'
+          element={
+            <GameEngineProvider>
+              <GameUI />
+            </GameEngineProvider>
+          }
+        />
+        <Route path='level-select' element={<LevelSelectPage />} />
+        {/* comment out when credits page is done */}
+        {/* <Route
+          path='credits'
+          element={
+            <>
+              <GlobalStyles />
+              <Credits />
+            </>
+          }
+        /> */}
```

Comment:
Page Credits czeka na CR, może uda się dziś zmerge'ować to od razu można by to odkomentować 😄

Reply:
Jasne, i dołożę też wtedy linkowanie do buttonu "homepage" w Credits.